PLUGINS.md

```diff
@@ -737,6 +740,91 @@ Examples of this are:
    
 ---
 
+---
+
+### Journal entry CQC
+New in version 5.1.3+
+```python
+def journal_entry_cqc(cmdr: str, is_beta: bool, entry: Dict[str, Any], state: Dict[str, Any]) -> None:
+    if entry['event'] == 'Location':
+        # We loaded to CQC match, lets detect map!
+        cqc_maps = {  # dict to map systems names to CQC maps
+        'Bleae Aewsy GA-Y d1-14': 'Asteria Point',
```

Comment:
The forum reference I know of for these is <https://forums.frontier.co.uk/threads/cqc-systems.234394/>, please add that to the comments here.

Reply:
But this thread doesn't contain such mapping, good news is I am posted it there.